OMCompiler/SimulationRuntime/c/simulation/solver/gbode_tableau.c

```diff
@@ -288,6 +288,26 @@ void getButcherTableau_MS(BUTCHER_TABLEAU* tableau)
   tableau->isKRightAvailable = TRUE;
 }
 
+// TODO: Describe me
```

Comment:
Could you add a super short description?
Maybe just link to Wiki: https://en.wikipedia.org/wiki/Heun%27s_method

Reply:
I added a link to the general Wikipedia page on RK methods. There will come more details on the references later, also for the other methods.